roofit/roofitcore/src/RooMinimizer.cxx

```diff
@@ -882,11 +810,10 @@ RooAbsMinimizerFcn *RooMinimizer::fitterFcn()
 /// to all RRV parameter representations and give this matrix instead of the
 /// HESSE matrix at the next save() call
 
-void RooMinimizer::applyCovarianceMatrix(TMatrixDSym& V)
+void RooMinimizer::applyCovarianceMatrix(TMatrixDSym const& V)
```

Comment:
It is not clear to me what is doing this function and what is the use case for this ? 
Should it belong to the Minimizer ?

Reply:
It's used to set the covariance matrix before computing errors with Minos, is cause you want to modify the covariance matrix before like in the `SumW2` error correction. It's easiest to have it in the RooMinimizer, because it has to be called between minimization and Minos, which are both covered by the RooMinimizer.